File: js/test/exists.test.js

const { test, describe } = require('node:test')
const { equal } = require('node:assert')


const { BluefinShieldconexSDK } = require('..')


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await BluefinShieldconexSDK.test()
    equal(null !== testsdk, true)
  })

})

File: dart/test/exists_test.dart
import 'harness.dart';

import '../lib/BluefinShieldconexSDK.dart';

void tests() {
  describe('exists', () {
    test('test-mode', (t) async {
      final testsdk = BluefinShieldconexSDK.test();
      equal(true, null != testsdk);
    });
  });
}
